atm.js

```diff
@@ -1,21 +1,87 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function(){
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+    $(".checkDepBtn").on("click", function(){
+      var deposit = parseInt($(".checkEntry").val());
+      var balance = $("#checkDisplay").text()
+      balance = parseInt(balance.replace("$", ""))
+      var checkingBalance = (deposit + balance)
+      $("#checkDisplay").text("$" + checkingBalance)
+    })
 
-}
+    $(".checkWdrwBtn").on("click", function(){
+      var withdrawal = parseInt($(".checkEntry").val())
+      var balance = $("#checkDisplay").text()
+      balance = parseInt(balance.replace("$", ""))
+      var checkingBalance = (balance - withdrawal)
+        if (checkingBalance >= 0) {
+          if (checkingBalance !== 0) {
+            $("#checkDisplay").text("$" + checkingBalance)
+            }
+          else {
+            $("#checkDisplay").text("$" + checkingBalance)
+            $("#checkDisplay").css("background-color", "#F52F4F");
```

Comment:
I recommend toggling a class here instead of setting an inline style. If you toggle a class, you can separate your concerns by keeping behavior in your JS files and styles in your CSS files.